packages/server-wallet/src/__test__/sync-objectives.test.ts

```diff
@@ -0,0 +1,79 @@
+import {CreateChannelParams} from '@statechannels/client-api-schema';
+import Knex from 'knex';
+
+import {defaultTestConfig, overwriteConfigWithDatabaseConnection} from '../config';
+import {DBAdmin} from '../db-admin/db-admin';
+import {seedAlicesSigningWallet, seedBobsSigningWallet} from '../db/seeds/1_signing_wallet_seeds';
+import {DBObjective, ObjectiveModel} from '../models/objective';
+import {Wallet} from '../wallet';
+import {createChannelArgs} from '../wallet/__test__/fixtures/create-channel';
+import {bob} from '../wallet/__test__/fixtures/participants';
+
+import {getChannelResultFor, getPayloadFor} from './test-helpers';
+
+jest.setTimeout(10_000);
+const aWalletConfig = overwriteConfigWithDatabaseConnection(defaultTestConfig(), {
+  database: 'TEST_A',
+});
+const bWalletConfig = overwriteConfigWithDatabaseConnection(defaultTestConfig(), {
+  database: 'TEST_B',
+});
+
+let a: Wallet;
+let b: Wallet;
+
+beforeAll(async () => {
+  await Promise.all([DBAdmin.dropDatabase(aWalletConfig), DBAdmin.dropDatabase(bWalletConfig)]);
+  await DBAdmin.createDatabase(aWalletConfig);
+  await DBAdmin.createDatabase(bWalletConfig);
```

Comment:
Is this copied from some other file? If so, seems like the wrong pattern to _create & drop_ some databases within multiple files.

Reply:
Yup, this was copied from [here](https://github.com/statechannels/statechannels/blob/master/packages/server-wallet/src/__test__/create-and-close-channel/direct-funding.test.ts)

Reply:
This probably should be a category of tests with its own `jest.config` that setups and tears down these databases.